ProgramowanieBot/Handlers/MessageHandler.cs

```diff
@@ -98,10 +99,7 @@ Task HandleReminderAsync()
                     return Task.CompletedTask;
 
                 var content = message.Content;
-                if (Config.PostResolveReminderKeywords.Any(k => content.Contains(k, StringComparison.InvariantCultureIgnoreCase)))
-                    return SendReminderMessageAsync();
-
-                return Task.CompletedTask;
+                return Config.PostResolveReminderKeywords.Any(k => content.Contains(k, StringComparison.InvariantCultureIgnoreCase)) ? SendReminderMessageAsync() : Task.CompletedTask;
```

Comment:
I considered this formatting before, but I think using `if` is clearer and looks better

Reply:
Okay. So be it.

Reply:
https://github.com/Programowanie-Discord/ProgramowanieBot/pull/5/commits/8cc2143743f4deb13c4bb18d4f53eaedc5564cde